package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "nest build",
-    "prestart": "setup-stargate default eddiehub",
+    "prestart": "setup-stargate c eddiehub",
+    "delete:keyspace": "setup-stargate d eddiehub",
```

Comment:
Oh awesome! Might be better to rename this to something like `stargate:keyspace:delete`

Reply:
Then could `prerestart` be renamed to `stargate:keyspace:create`, then `start:dev` could be updated to 

```
"start:dev": "npm stargate:keypspace:create && nest start --watch",
``